xDscResourceDesigner.Tests.ps1

```diff
@@ -44,6 +44,18 @@ end
                 $result | Should Be $true
             }
         }
+
+        Context 'A resource with both required files, but schema mof has encoding UTF8 BOM' {
+            Setup -Dir TestResource
+            Setup -File TestResource\TestResource.psm1 -Content (Get-TestDscResourceModuleContent)
+            Get-TestDscResourceSchemaContent |
+                Out-File -Encoding utf8 -FilePath (Join-Path -Path $TestDrive -ChildPath 'TestResource\TestResource.schema.mof') -Force
+
+            It 'Should write out the correct error to console, and pass the test' {
```

Comment:
How are you checking that the correct error was output? And it looks like it did not pass Test-xDscResource if it is supposed to return false?

Reply:
My bad, should have mocked Write-Error, don't know what I was thinking 😃 And I think the description is better now. 
I would have rather used the localized data string, but then I would have to rewrite some of the tests to get the context blocks inside an `InModuleScope`. I will do that if you like.